CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Change Log
 
+## [styx-0.7.7](https://github.com/HotelsDotCom/styx/tree/styx-0.7.7) (2018-05-17)
+[Full Changelog](https://github.com/HotelsDotCom/styx/compare/styx-0.7.6...styx-0.7.7)
+
+**Implemented enhancements:**
+
+- Config validator: Support for MAP types [\#143](https://github.com/HotelsDotCom/styx/issues/143)
+- Config validator: Improve field type detection [\#142](https://github.com/HotelsDotCom/styx/issues/142)
```

Comment:
As config validator is not yet activated in Styx core, I am thinking that these enhancements should not be listed here as they are invisible for the end user.

What do you think?

Reply:
Makes sense.

Reply:
Change pushed.